runtime/lua/vim/shared.lua

```diff
@@ -374,7 +374,7 @@ end
 ---
 ---@see |extend()|
 ---
----@param behavior string Decides what to do if a key is found in more than one map:
+---@param behavior "error"|"keep"|"force" (string) Decides what to do if a key is found in more than one map:
```

Comment:
```suggestion
---@param behavior 'error'|'keep'|'force' Decides what to do if a key is found in more than one map:
```

Reply:
Should [this line](https://github.com/neovim/neovim/blob/f24ffa0b80ad03e7e986f1114fc26cfb4e1d6487/runtime/lua/vim/shared.lua#L393) be changed similarly?

Reply:
Sure